book/website/reproducible-research/reviewing/reviewing-motivation.md

```diff
@@ -59,3 +80,8 @@ Peer-to-peer review creates two-way exchange of information across a web strung
 
 Reviews conducted in the right spirit (see especially {ref}`here<rr-reviewing-recommendation-be-nice>`) also serve an important purpose in bringing team members together and creating group cohesion.
 In particular, good reviews by core team members of the work of newcomers to a project can help make those newcomers feel welcomed and valued, and encourage their continued participation.
+
+For some projects it may even make sense to have code review as a step in the onboarding process, having contributors perform a certain number of reviews before they're invited to contribute in other areas.
```

Comment:
Could you elaborate more on why people might review first before contributing code?

```suggestion
For some projects it may even make sense to have code review as a step in the onboarding process, having contributors perform a certain number of reviews before they are invited to contribute in other areas.
```

Reply:
Sure, I'll try to add some justification for this.